README.md

```diff
@@ -182,6 +182,12 @@ Once the image is in your repo you can install it into your cluster from your re
 envsubst < deployment/private-installer.yaml | kubectl apply -f -
 ```
 
+
+### Fully Private Cluster Considerations
```

Comment:
We do not plan on adding every general EKS setup recommendation. If we make this a generic requirement in the "Requirements" section that states the cluster needs access to the Secrets Manager endpoint, we would consider merging that.